sandbox-templates/react-markdown-with-create-react-app/src/app.js

```diff
@@ -0,0 +1,20 @@
+// ⚠️ Important! Please make sure the dependencies are up to date.
+// You can refresh them in the Dependencies section (left-bottom on CodeSandbox)
```

Comment:
with stackblitz also being linked should this comment be updated?
suggestions on new copy?